src/routes/Home/components/Home.tsx

```diff
@@ -42,6 +43,7 @@ public onClick=()=> {
     const {projectLength} = this.props
   return (
     <div className="row home-container">
+      <FixMeMetas title="Fixme" description="home page meta description"/> 
```

Comment:
The description isn't really descriptive, maybe something like "FixMe helps you find contribution opportunities across multiple open source projects" (or something similar) would be clearer